### ISSUE TITLE
Use net/http/httputil.ReverseProxy instead to support WebSockets

### ISSUE THREAD

**Issue by ncpa0cpl** [domcloud/proxy-fix#1]
Current implementation does not support websockets, but this can be easily fixed by simply using `net/http/httputil.ReverseProxy` from the Go standard library. Here's how it could look like: https://github.com/ncpa0cpl/proxy-fix/blob/main/main.go

I've also tested it on a real Dom Cloud instance, it is correctly serving both `https` and `wss` connections from a Bun server: https://camqiwa.fra.dom.my.id/

**Comment by willnode** [comment]
Thank you for this suggestion!  🙏  I'll incorporate your idea to this app soon 👍

**Comment by willnode** [comment]
We've fixed our `bunfix` implementation to support websocket  🎉  Thanks for pointing out @ncpa0cpl !

To update `bunfix` you can run this in SSH:

```
(cd ~/.local/bin/; wget -qO- https://github.com/domcloud/proxy-fix/releases/download/v0.1.2/proxy-fix-linux-amd64.tar.gz | tar xz && mv proxy-fix-linux-amd64 bunfix)
```
